src/provider/types.ts

```diff
@@ -26,13 +32,25 @@ export interface BitcoinProvider extends BaseBitcoinProvider {
   getCapabilities?: (request: string) => Promise<GetCapabilitiesResponse>;
 }
 
+export interface WebbtcProvider {
+  id: string;
+  name: string;
+  icon: string;
+  webUrl?: string;
+  chromeWebStoreUrl?: string;
+  mozillaAddOnsUrl?: string;
+  googlePlayStoreUrl?: string;
+  iOSAppStoreUrl?: string;
+  methods?: string[];
+}
+
 declare global {
   interface XverseProviders {
     BitcoinProvider?: BitcoinProvider;
   }
-
   interface Window {
     BitcoinProvider?: BitcoinProvider;
-    XverseProviders?: XverseProviders
+    XverseProviders?: XverseProviders;
+    webbtc_providers: WebbtcProvider[];
```

Comment:
This should be optional field in case there are no wallets.
```suggestion
    webbtc_providers?: WebbtcProvider[];
```